libraries/AP_Motors/examples/AP_Motors_test/AP_Motors_test.cpp

```diff
@@ -74,7 +74,7 @@ void setup()
     // set rc channel ranges
     rc1.set_angle(4500);
     rc2.set_angle(4500);
-    rc3.set_range(130, 1000);
+    rc3.set_range(1000);
```

Comment:
the bottom end of the range was only used in 2 places. One was in the examples, the other was in copter tuning. I removed the bottom end and made it a special case for copter tuning. That simplified the logic in RC_Channel